src/mpid/ch4/shm/ipc/src/ipc_init.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "mpidimpl.h"
+#include "ipc_noinline.h"
+#include "ipc_control.h"
+#include "shm_control.h"
+
+static void regist_shm_ctrl_cb(void)
```

Comment:
```suggestion
static void register_shm_ctrl_cb(void)
```

Reply:
Please replace `regist` with `register` and `deregist` with `deregister` everywhere.